client/keys/add.go

```diff
@@ -77,9 +77,6 @@ the flag --nosort is set.
 	cmd.Flags().Uint32(flagIndex, 0, "Address index number for HD derivation")
 	cmd.Flags().String(flags.FlagKeyAlgorithm, string(hd.Secp256k1Type), "Key signing algorithm to generate keys for")
 
-	cmd.SetOut(cmd.OutOrStdout())
-	cmd.SetErr(cmd.ErrOrStderr())
-
```

Comment:
Ditto

Reply:
OK. Just make sure that default behaviour is not affected :+1: i.e. test manually, please

Reply:
> OK. Just make sure that default behaviour is not affected 👍 i.e. test manually, please

I went through and manually tested, as well as audited all the uses of `cmd.Print*`.  That's what https://github.com/cosmos/cosmos-sdk/pull/8628/commits/8c78b32ba21bc63e6056b909cd23559feb0b8fbf is the result of.